lib/Babel.js

```diff
@@ -53,15 +53,16 @@ function Babel(uri, callback) {
         } else {
           self.languageMap = {};
         }
+
         self.langSplicers = {};
         self.pickLanguage = true;
         self.splicer = self._createSplicer(query.defaultLanguage);
         break;
 
       case 'json2tags:':
-        if (query.defaultLanguage || query.languageMap) {
+        if (query.defaultLanguagee || query.languageMap) {
           throw new Err('defaultLanguage and languageMap params ' +
-            'are not allowed for "json2tags://" protocol');
+            'is not allowed for "json2tags://" protocol');
```

Comment:
is?

Reply:
Meh, I originally took it out and then brought it back and on the way messed up the sentence...